src/components/main-page/Location/LocationMap.tsx

```diff
@@ -0,0 +1,34 @@
+import { MapContainer, Marker, TileLayer } from "react-leaflet";
+import "leaflet/dist/leaflet.css";
+import styles from "./LocationMap.module.scss";
+import { LatLngExpression } from "leaflet";
+import markerIconPng from "leaflet/dist/images/marker-icon.png";
+import { Icon } from "leaflet";
+
+const position: LatLngExpression = [52.404556560532235, 16.949742396468974];
+
+const LocationMap = () => {
+  return (
+    <MapContainer
+      center={position}
+      zoom={15}
+      scrollWheelZoom={false}
+      className={styles.map}
+    >
+      <TileLayer
+        attribution='&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
+        url="https://mt1.google.com/vt/lyrs=m&x={x}&y={y}&z={z}"
+      />
+      <Marker
+        position={position}
+        icon={
+          new Icon({
+            iconUrl: markerIconPng.src,
+          })
+        }
+      ></Marker>
```

Comment:
When without children, component can be used as self-closing tag
```suggestion
      />
```